### ISSUE TITLE
Revised Keras result (3 model)

### ISSUE THREAD

**Issue by sukitpom** [TheeranatSri/DADS7202_project_1#2]
Keras basic parameter

![basic model](https://user-images.githubusercontent.com/33378401/189479365-c8c48931-b67a-49a7-a620-22ab6980f7ac.png)

**Comment by sukitpom** [comment]
Accuracy of train data set from keras basic parameter

![basic train](https://user-images.githubusercontent.com/33378401/189479373-df333a86-ec6b-47db-b5be-67c16a4eaf51.png)

**Comment by sukitpom** [comment]
Auto hyperparameter from keras tunner

![auto hyperparameter model](https://user-images.githubusercontent.com/33378401/189479380-1a52c704-5bd8-4474-b250-2a1b8f35f248.png)

**Comment by sukitpom** [comment]
Accuracy of train data set from keras auto hyperparameter tunner

![auto hyperparameter train](https://user-images.githubusercontent.com/33378401/189479390-40d52d9a-3a13-4cc3-91f3-21333e3c1542.png)

**Comment by sukitpom** [comment]
![manual hyperparameter model](https://user-images.githubusercontent.com/33378401/189479397-492fb3bd-a34d-47e9-b2b0-418ba4264c1b.png)

**Comment by sukitpom** [comment]
![manual hyperparameter train](https://user-images.githubusercontent.com/33378401/189479408-700980f0-9ade-4dfa-9df8-cdd545e6d082.png)

**Comment by sukitpom** [comment]
Keras manual hyperparameter

![manual hyperparameter model](https://user-images.githubusercontent.com/33378401/189479752-4fc352d0-04cc-44d2-8626-cb96a7eda86d.png)

**Comment by sukitpom** [comment]
Accuracy of train data set from keras manual hyperparameter

![manual hyperparameter train](https://user-images.githubusercontent.com/33378401/189479794-337b9737-ecf0-4edd-ab08-1a7c269bf31d.png)

**Comment by sukitpom** [comment]
Summary result of MLP

![image](https://user-images.githubusercontent.com/33378401/189479945-10f21630-4881-4c5a-9b50-276d80e06e7a.png)
